packages/relay/src/services/UserService.ts

```diff
@@ -131,14 +131,13 @@ export class UserService {
             ]
         )
 
-        const parsedLogs = await TransactionManager.executeTransaction(
+        const { logs, txHash } = await TransactionManager.executeTransaction(
```

Comment:
After modifing callContract method of TransactionManager, user sign up function can be modified to one line like this `TransactionManager.callContract('userSignUp',[signupProof.publicSignals, signupProof.proof, hashUserId, fromServer], TXEXECUTION.EXECUTE)`